.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.degitx.simulator"
+version = "0.0.1"
+
+val kotlinVersion = "1.4.10"
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven { url = uri("https://kotlin.bintray.com/kotlinx") }
+}
+
+plugins {
+    java
+    kotlin("jvm") version "1.4.10"
+    kotlin("plugin.serialization") version "1.4.10"
+    application
+}
+
+val compileKotlin: KotlinCompile by tasks
+val compileTestKotlin: KotlinCompile by tasks
+compileKotlin.run {
+    kotlinOptions.jvmTarget = "1.8"
+}
+compileTestKotlin.run {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion")
+    implementation("org.jetbrains.kotlinx:kotlinx-cli:0.3")
+
+    testImplementation("org.testng:testng-6.14.3")
+}
+
+tasks.test {
+    // enable TestNG support (default is JUnit)
+    useTestNG()
```

Comment:
@IlyaMoskva why testNG, not Junit?

Reply:
@g4s8 Why not TestNG? They are very similar, so choice based on this comparison: https://www.guru99.com/junit-vs-testng.html.

Reply:
@IlyaMoskva ok, let's use testNG if you think it's better here